x/ccv/consumer/keeper/soft_opt_out.go

```diff
@@ -73,3 +83,42 @@ func (k Keeper) GetSmallestNonOptOutPower(ctx sdk.Context) int64 {
 	}
 	return int64(binary.BigEndian.Uint64(bz))
 }
+
+func (k Keeper) UpdateSlashingSigningInfo(ctx sdk.Context) {
+	smallestNonOptOutPower := k.GetSmallestNonOptOutPower(ctx)
+
+	// Update SigningInfo for opted out validators
+	valset := k.GetAllCCValidator(ctx)
+	// sort validators by power ascending
+	sort.SliceStable(valset, func(i, j int) bool {
```

Comment:
valset should be ordered already by validator address (as claimed by SDK)
pls check if the re-ordering according to power is still needed as it comes with an overhead.

Reply:
Nice catch. See https://github.com/cosmos/interchain-security/pull/1549/commits/b00c874094af6b725fe1d0d97e5ee0a89e3e50eb